proto/mixer.proto

```diff
@@ -197,19 +197,39 @@ message GetRelatedLocationsResponse {
   string payload = 1;
 }
 
+message Place {
+  string dcid = 1;
+  string name = 2;
+  int32 pop = 3;
+}
+
+message Places {
+  repeated Place places = 1;
+}
+
 // Request to get all stat data in landing page for a place.
 message GetLandingPageDataRequest {
   // The dcid of the place.
   string place = 1;
-  // A list of all stat vars used in the landing page.
+  // A list of additional stat vars need to be fetched in addition to the
+  // data in cache. This is to be used in local development, where new chart is
+  // added to chart config but data is not added to cache (delay from cache
+  // build).
   repeated string stat_vars = 2;
```

Comment:
Call it extra_stat_vars or test_stat_vars ?

Also, should we reserve 2 and use tag 3?

Reply:
or new_stat_vars?